### PR TITLE
Achievements carousel on homepage

### PR DESCRIPTION
**Pull request by BhavyaSheth22**
Added a carousel setup in the home page to showcase achievements by members. This is a view using some dummy data.

ezgif com-gif-maker

TODO:

- Add achievements description to schema and UI 
- Possibly have a different page for showing detailed achievement info

### CODE REVIEW

File: new_client/src/components/Home/components/Achievements/Achievements.jsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+});
+
+const Achievements = (props) => {
+  const [achievements, setAchievements] = useState([]);
+
+  useEffect(() => {
+    axios.get(process.env.REACT_APP_API + "/achievements").then((res) => {
+      setAchievements(res.data["achievements"]);
+    });
+  }, [achievements]);
```
Comment (ShubhankarKG) [inline new_client/src/components/Home/components/Achievements/Achievements.jsx:37]: We want this useEffect to run only once, so achievements as a dependency is not needed.

**Review by ShubhankarKG — APPROVED** [review]
Thanks, rest of the stuff looks good. Just a small fix.

**Comment by BhavyaSheth22** [comment]
I've made the requested changes to the useEffect hook. I've also added the description attribute to the schema and the UI. Finally, I made the view project option in the card optional as the schema doesn't always require it.

Screenshot from 2021-02-04 09-48-49

**Comment by ShubhankarKG** [comment]
i added two changes to this:

1. fixed a padding issue for the Our Mission.
2. updated the Learn more to scroll to  next div

**Pull request merged**